hazelcast/serialization/compact.py

```diff
@@ -1769,6 +1774,11 @@ def _init(self):
             else:
                 fix_sized_fields.append(field)
 
+        # Fix sized fields should be in descending order of size in bytes.
```

Comment:
Why are they sorted by size in bytes?

Reply:
That was a requirement of the wire-level format https://hazelcast.atlassian.net/wiki/spaces/IMDG/pages/3507487040/Compact+Wire-level+Format+Design+Document#Fixed-Size-Fields